packages/x-components/src/components/filters/labels/__tests__/base-price-label.spec.ts

```diff
@@ -8,17 +9,26 @@ function renderBasePriceLabel({
   format,
   lessThan = 'Less than {max}',
   from = 'More than {min}',
-  fromTo = 'From {min} to {max}'
-}: RenderBasePriceLabelOptions): Wrapper<BasePriceFilterLabel> {
-  return mount(BasePriceFilterLabel, {
-    propsData: {
-      filter,
-      format,
-      lessThan,
-      from,
-      fromTo
+  fromTo = 'From {min} to {max}',
+  template = '<BasePriceFilterLabel :filter="filter" :format="format" ' +
+    ':less-than="lessThan" :from="from" :from-to="fromTo" />'
```

Comment:
We're not modifying the template in any test, so you can move this inside the `mount` directly and remove it from the parameters.